src/apple/config/raw.rs

```diff
@@ -42,10 +42,44 @@ impl Display for PromptError {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq)]
+#[serde(untagged)]
+pub enum OneOrMany<T: Debug> {
```

Comment:
I could see this abstraction being useful elsewhere in the future, but for now I guess it's fine to keep it here where we actually use it.

Reply:
I think this case is clear-cut enough that it makes sense to proactively move to `util`